units/src/amount/signed.rs

```diff
@@ -144,12 +144,11 @@ impl SignedAmount {
     #[allow(clippy::missing_panics_doc)]
     pub const fn from_int_btc_const(whole_bitcoin: i16) -> SignedAmount {
         let btc = whole_bitcoin as i64; // Can't call `into` in const context.
-        match btc.checked_mul(100_000_000) {
-            Some(amount) => match SignedAmount::from_sat(amount) {
-                Ok(amount) => amount,
-                Err(_) => panic!("unreachable - 65536 BTC is within range")
-            }
-            None => panic!("unreachable - mul by 65536 cannot overflow a i64"),
+        let sats = btc * 100_000_000;
```

Comment:
Did you mean to do this in previous commit?

Reply:
Bother, my bad. Thanks man